examples/siro_sandbox/controllers/fetch_baselines_controller.py

```diff
@@ -430,7 +430,7 @@ def act(self, obs, env):
 
         if self.current_state == FetchState.WAIT:
             self.current_state = FetchState.FOLLOW
-            self._init_policy_input()
+            self._policy_info = self._init_policy_input()
```

Comment:
How about rename `_init_policy_input` to `_get_policy_info`?

Reply:
done!